packages/darklang/cli/selfUpdate.dark

```diff
@@ -10,105 +17,281 @@ module Darklang =
         { tag_name: String
           assets: List<Asset> }
 
+      let defaultConfig =
+        Dict
+          { version = "v0.0.0"
+            lastUpdateTimestamp = "0" }
+
+
+      let getFromGithub
+        (bearerToken: String)
+        (path: String)
+        : Stdlib.Result.Result<String, String> =
+        let headers =
+          [ ("authorization ", "bearer" ++ bearerToken)
+            ("accept", "application/vnd.github+json")
+            ("user-Agent", "Darklang") ]
+
+        let response = Darklang.Stdlib.HttpClient.request "GET" path headers []
```

Comment:
I didn't use the Stdlib.HttpClient.get wrapper function here because it only takes a uri as an arg (headers are empty by default)